test/commands/scorecard.test.js

```diff
@@ -116,8 +117,10 @@ test("should support github only", async() => {
       ].join("\n")
     }
   });
-  const result = testingModule.getCurrentRepository();
+  const logs = [];
+  console.log = (str) => logs.push(str);
```

Comment:
Maybe add a `NOTE` to explain why we do this here ^^

It's pretty smart tho, I learn something, thanks a lot @PierreDemailly ^^

Reply:
ahah, yeah it's quite useful but i'm not sure if this is a good practice tbh :s 
I'm adding a note ;)

Reply:
Can you confirm wether my `NOTE` is good enough ? :)

Reply:
Good on my side ^^